<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KismetGroupEditWindow</class>
 <widget class="QWidget" name="KismetGroupEditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>事件组属性</string>
  </property>
  <property name="windowIcon">
   <iconset resource="orion2.qrc">
    <normaloff>:/images/unique32.png</normaloff>:/images/unique32.png</iconset>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>透明度</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="alphahorizontalSlider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="alphalabel">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>文本</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="colorhorizontalSlider">
      <property name="maximum">
       <number>359</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="renamelineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>填充色</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="PictureWidget" name="previewwidget" native="true">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="activecheckBox">
      <property name="text">
       <string>启动时激活</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="sortpushButton">
      <property name="text">
       <string>次序</string>
      </property>
      <property name="icon">
       <iconset resource="orion2.qrc">
        <normaloff>:/images/script.png</normaloff>:/images/script.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okpushButton">
      <property name="text">
       <string>确定</string>
      </property>
      <property name="icon">
       <iconset resource="orion2.qrc">
        <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelpushButton">
      <property name="text">
       <string>取消</string>
      </property>
      <property name="icon">
       <iconset resource="orion2.qrc">
        <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PictureWidget</class>
   <extends>QWidget</extends>
   <header>picturewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="orion2.qrc"/>
 </resources>
 <connections/>
</ui>
